### DISCUSSION TITLE
Add Assistants Create Support

### DISCUSSION THREAD

**Discussion by elvis-cai** [BerriAI/litellm#4618] [General]
Hi Team
From doc https://litellm.vercel.app/docs/assistants seems it only supports litellm SDK or restapi, just wondering if there's any place to support openai package natively? 
had a test locally, get openai.APIStatusError: Error code: 405 - {'detail': 'Method Not Allowed'}

```
from openai import OpenAI

client = OpenAI(base_url="https://litellmui.testdomain.com")

assistant = client.beta.assistants.create(
    name="Math Tutor",
    instructions="You are a personal math tutor. Write and run code to answer math questions.",
    tools=[{"type": "code_interpreter"}],
    model="gpt-4o"
)
```
Thank you

**Comment by ishaan-jaff** [comment]
hi @elvis-cai - planning on adding support for this today

**Reply by ishaan-jaff** [reply]
hi @elvis-cai this is now fixed. You can make the following call to litellm proxy

Will be live on **LiteLLM 1.41.15**

```
from openai import OpenAI

client = OpenAI(base_url="https://litellmui.testdomain.com")

assistant = client.beta.assistants.create(
    name="Math Tutor",
    instructions="You are a personal math tutor. Write and run code to answer math questions.",
    tools=[{"type": "code_interpreter"}],
    model="gpt-4o"
)
```

**Reply by elvis-cai** [reply]
perfect, thanks @ishaan-jaff  for the quick fix, will test it when 1.41.15 is available  👍

**Comment by ishaan-jaff** [comment]
Hey @elvis-cai  Can we setup a 1:1 support channel? Would like to improve litellm for your use-case 

[LinkedIn](https://www.linkedin.com/in/reffajnaahsi/)
[Discord](https://discord.com/invite/wuPM9dRgDw) (just 👋 wave on #general)

**Reply by elvis-cai** [reply]
it will be great, not sure if Slack is available as well.

**Reply by ishaan-jaff** [reply]
slack works - what's the best email to send an invite to @elvis-cai ?

**Reply by elvis-cai** [reply]
elvis.cai@airwallex.com,  I could create a Slack connect in our org and invite you and my colleagues together if it's easier.

**Reply by ishaan-jaff** [reply]
@elvis-cai - slack invite sent just now

 If you prefer inviting me my email is `ishaanjaffer0324@gmail.com`

**Reply by elvis-cai** [reply]
all good, joined
